### PR TITLE
Feautre/test option

### PR DESCRIPTION
**Pull request by dmitmel**
See #24

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -14,6 +14,7 @@
   "rules": {
     "semi": [2, "never"],
     "comma-dangle": [2, "always-multiline"],
+    "no-param-reassign": 0,
```
Comment (diegohaz) [inline .eslintrc]: Let's try to keep methods pure and not mutate arguments.

File: src/index.js [hunk 2]
```diff
@@ -8,20 +8,20 @@ import type { Options } from './types'
...
-const splitVendor = (options: string | Options): Function => {
-  const {
-    name = 'vendor',
-    exclude,
-  } = typeof options === 'object' ? options : { name: options, exclude: undefined }
+const splitVendor = (nameOrOptions: string | Options): Function => {
+  const options: Options = typeof nameOrOptions === 'object' ? nameOrOptions : { name: nameOrOptions }
+  if (options.name == null) options.name = 'vendor'
+  /* istanbul ignore next */
+  if (options.test == null) options.test = /\.js$/
```
Comment (diegohaz) [inline src/index.js]: Don't mutate arguments and let's not ignore coverage. It's there for a reason, I can write tests later if you can't.

**Review by diegohaz — CHANGES_REQUESTED** [review]
Thank you so much, @dmitmel. I've made some comments. Let's try to keep code quality.

**Comment by diegohaz** [comment]
Thank you, @dmitmel.

**Comment by dmitmel** [comment]
@diegohaz Would you like to create a new release with changes made in this PR?

**Pull request merged**